### PR TITLE
Update the Minimum Required iOS Version to 10.0

### PR DESCRIPTION
**Pull request by joeljfischer**
This proposal is to update the minimum supported iOS deployment version from iOS 8.0 to iOS 10.0.

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech This proposal is ready for review

File: proposals/nnnn-ios-update-minimum-version-ios10.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Update the Minimum Required iOS Version to 10.0
+* Proposal: [SDL-NNNN](nnnn-ios-update-minimum-version-ios10.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / HMI / Policy Server / SHAID / RPC / Protocol]
```
Comment (theresalech) [inline proposals/nnnn-ios-update-minimum-version-ios10.md:5]: Is this accurate?  I only see iOS described throughout the proposal.

File: proposals/nnnn-ios-update-minimum-version-ios10.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Introduction
+This proposal is to update the minimum supported iOS deployment version from iOS 8.0 to iOS 10.0.
+
+## Motivation
+In September, Xcode 12 will be released. In the current beta release notes, it's noted that Xcode 12 drops support for creating apps below iOS 9.0. We currently support down to iOS 8.0. Therefore, we will need to update our minimum supported version to at least iOS 9.0.
```
Comment (theresalech) [inline proposals/nnnn-ios-update-minimum-version-ios10.md:11]:
```suggestion
In September 2020, Xcode 12 will be released. In the current beta release notes, it's noted that Xcode 12 drops support for creating apps below iOS 9.0. We currently support down to iOS 8.0. Therefore, we will need to update our minimum supported version to at least iOS 9.0.
```

File: proposals/nnnn-ios-update-minimum-version-ios10.md (hunk 1 above)
Comment (joeljfischer) [reply proposals/nnnn-ios-update-minimum-version-ios10.md:5]: No, ha, I somehow missed that line

**Comment by joeljfischer** [comment]
@theresalech thanks for the review, it's ready for your review again.

**Pull request merged**